presto-orc/src/main/java/com/facebook/presto/orc/OrcReader.java

```diff
@@ -490,4 +492,23 @@ public OrcDataSource getOrcDataSource()
     {
         return orcDataSource;
     }
+
+    public StripeFooter readStripeFooter(StripeInformation stripe)
```

Comment:
will this only be called from test or do you see use cases where this will be called in prod as well.

Reply:
Prod - no. This will be mostly used in tools where we need metadata about the file. Right now we have footer information + basic stripe info, this change will close the gap by adding stripe footers.